common/journal_entries/00_sick_man.txt

```diff
@@ -0,0 +1,1007 @@
+		}
+		trigger_if = {
+			limit = {
+				has_variable = liberal_tanzimat
+			}
+			OR = {
+				has_law = law_type:law_private_schools
+				has_law = law_type:law_public_schools
+			}
+		}
+		trigger_if = {
+			limit = {
+				has_variable = absolute_tanzimat
+			}
+			OR = {
+				has_law = law_type:law_patriot_schools
+				has_law = law_type:law_religious_schools
+			}
+		}
+		scope:journal_entry = {
+			is_goal_complete = yes
+		}
+	}
+	on_complete = {
+		if = {
+			limit = {
+				has_variable = liberal_tanzimat
+			}
+			add_modifier = {
+				name = sick_man_liberal_education
+				years = 10
+			}
+		}
+		else = {
+			add_modifier = {
+				name = sick_man_absolute_education
+				years = 10
+			}
+		}
+		set_variable = completed_sick_man_education_je
+		custom_tooltip = je_sick_man_advance
+		change_variable = {
+			name = sick_man_var
+			add = 1
+		}
+	}
+	current_value = {
+		value = root.literacy_rate
+	}
+	goal_add_value = {
+		add = 0.15
+	}
+	invalid = {
+		custom_tooltip = {
+			text = finished_sick_man_tt
+			has_variable = sick_man_complete
+		}
+	}
+	transferable = no
+	weight = 5000
+	progressbar = yes
+	remove_modifier = imbalanced_education
+}
+
+# Liberalism
+je_sick_man_liberal_voting = {
+	icon = "gfx/interface/icons/event_icons/event_election.dds"
+	immediate = {
+		set_variable = {
+			name = sick_man_voting_liberal_var
+			value = 0
+		}
+	}
+	complete = {
+		scope:journal_entry = {
+			is_goal_complete = yes
+		}
+		OR = {
+			has_law = law_type:law_right_of_assembly
+			has_law = law_type:law_protected_speech
+		}
+		OR = {
+			has_law = law_type:law_wealth_voting
+			has_law = law_type:law_census_voting
+			has_law = law_type:law_universal_suffrage
+		}
+	}
+	on_complete = {
+		custom_tooltip = je_sick_man_advance
+		change_variable = {
+			name = sick_man_var
+			add = 1
+		}
+		add_loyalists = {
+			value = 0.1
+			strata = poor
+		}
+		add_radicals = {
+			value = 0.15
+			strata = rich
+		}
+		add_modifier = {
+			name = sick_man_liberal_voting
+			years = 10
+		}
+	}
+	invalid = {
+		custom_tooltip = {
+			text = finished_sick_man_tt
+			has_variable = sick_man_complete
+		}
+	}
+	current_value = {
+		value = var:sick_man_voting_liberal_var
+	}
+	goal_add_value = {
+		add = 120
+	}
+	on_monthly_pulse = {
+		effect = {
+			if = {
+				limit = {
+					OR = {
+						has_law = law_type:law_right_of_assembly
+						has_law = law_type:law_protected_speech
+					}
+					OR = {
+						has_law = law_type:law_wealth_voting
+						has_law = law_type:law_census_voting
+						has_law = law_type:law_universal_suffrage
+					}
+				}
+				change_variable = {
+					name = sick_man_voting_liberal_var
+					add = 1
+				}
+			}
+		}
+	}
+	transferable = no
+	weight = 5000
+	progressbar = yes
+}
+
+# Secularism
+je_sick_man_liberal_secularism = {
+	icon = "gfx/interface/icons/event_icons/event_scales.dds"
+	complete = {
+		scope:journal_entry = {
+			is_goal_complete = yes
+		}
+		has_law = law_type:law_guaranteed_liberties
```

Comment:
There is a separate tanzimat already requiring right of assembly. This one should be on par in difficulty with the Egypt tanzimat for Absolutism as these are the two options which remove internal strife which is the most intractable modifier. since the only other real requiirement is to pass total separation, this should be feasible given everything.